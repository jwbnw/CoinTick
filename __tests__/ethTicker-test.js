import React from "react";
import EthTicker from "./../components/ethTicker";

import renderer from "react-test-renderer";

test("Renders Correctly", () => {
  const tree = renderer.create(<EthTicker />).toJSON();
  expect(tree).toMatchSnapshot();
});
