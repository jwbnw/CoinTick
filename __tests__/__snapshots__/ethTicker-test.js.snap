// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignSelf": "center",
      "opacity": 1,
      "padding": 10,
    }
  }
>
  <Image
    source={Object {}}
  />
</View>
`;
